docs/large_cohort_pipeline_readme.md

```diff
@@ -0,0 +1,235 @@
+# [Large Cohort Pipeline](https://github.com/populationgenomics/production-pipelines)
+## Quick Overview
+This pipeline takes in FASTQ or CRAM files as input and runs a series of python scripts to process all files into a final annotated, joint-called [sparse matrix table](https://hail.is/docs/0.2/vds/index.html ""). Specifically, the pipeline takes on the following stages:
+1. CramMultiQC Stage:  Aligns the data and generates a MultiQC report
+2. GvcfMultiQC Stage: Generates GVCFs and generates a MultiQC report
+3. FrequenciesStage: Runs the [VDS combiner](https://hail.is/docs/0.2/vds/hail.vds.combiner.VariantDatasetCombiner.html#hail.vds.combiner.VariantDatasetCombiner "") with GVCFs as an input to produce a VariantDataset (VDS), then performs sample-level QC (pedigree, sex and ancestry inference)
+4. LoadVqsr Stage: Converts the VDS into a sites-only VCF-ready table, then performs allele-specific VQSR variant QC.
+
+Graphically, the pipeline has the following workflow:
+
+![uml](large_cohort.png)
+
+## Running the pipeline
+To run, the pipeline takes in at least two config files: an exome/genome config file, specifying whether samples are whole genome or whole exome sequences (e.g., `configs/genome.toml`), and a config file specifying parameters specific to the dataset. This second config file **must** be named after the dataset + access level, e.g., `configs/bioheart-test.toml` . By default, a third config file,  `configs/defaults/large_cohort.toml` , is also loaded into the pipeline. This config file does (==x, y, z - how is this different to the dataset + access level config file?==). Any additional config files can be provided by adding  `--config` to the analysis runner command. 
```

Comment:
@vladsavelyev just curious how the dataset + access level config file is differnet to the larcoh file? I'm not really sure which one feeds into which parts of the script

Reply:
There are no limitations on names and number of configs. You can create only one config file and call it `myconfig.toml`, add all required fields there including `sequencing_type`. Or you can split it up, it's just the matter of convenience. E.g. current [prophecy](https://github.com/populationgenomics/production-pipelines/blob/main/configs/prophecy.toml) and [bioheart](https://github.com/populationgenomics/production-pipelines/blob/main/configs/bioheart-test.toml) configs contain all parameters needed to run the large-cohort pipeline, including `sequencing_type = 'genome'`.

Reply:
`configs/defaults/large_cohort.toml` contains the defaults, applicable to the workflow in general, regardless of a dataset. You don't need to worry about it unless you want some reference to override certain parameters in your run-specific configs, like e.g. `bioheart.toml`

Reply:
got it, thank you!